vision_agent/utils/sim.py

```diff
@@ -133,6 +133,12 @@ def check_load(
         df: pd.DataFrame,
     ) -> bool:
         load_dir = Path(load_dir)
+        if (
+            not Path(load_dir / "df.csv").exists()
+            or Path(load_dir / "embs.npy").exists()
```

Comment:
❔ Not sure about the intention here, if the function shoul only returns `False` if either required file is missing, let's update it as
```suggestion
            or not Path(load_dir / "embs.npy").exists()
```

otherwise if `embs.npy` exists, it will incorrectly return `False`.

Reply:
good catch!